.github/workflows/ci-instrumentation-canary.yml

```diff
@@ -59,6 +59,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
+      - name: "Test Ruby 3.3"
+        # BLOCKED BY: https://github.com/bigcommerce/gruf/pull/197
```

Comment:
Hey @arielvalentin - this has now been addressed and released, in connection with the grpc 1.61 release: https://rubygems.org/gems/gruf/versions/2.19.0